### PR TITLE
Init Docker environment

### PR DESCRIPTION
**Pull request by vincentchalamon**
- [x] Update documentation
- [x] Enable default `docker-compose.yaml` file (can be customized through `docker-compose.yml` file)
- [x] Check Nginx & PHP-FPM configuration

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,35 @@ A demo application (a bookstore) is pre-installed.
 * Build your first custom client using Javascript, CORS headers are already
```
Comment (dunglas) [inline README.md]: Can you add this block in this repo instead? https://github.com/dunglas/api-platform-doc

Comment (vincentchalamon) [inline README.md]: Done

**Comment by vincentchalamon** [comment]
@dunglas [Docker container](https://registry.hub.docker.com/u/vincentchalamon/symfony/) updated running through debian:jessie with PHP 5.6

**Comment by dunglas** [comment]
Thank you @vincentchalamon

**Pull request merged**